scss/components/_button.scss

```diff
@@ -100,7 +101,7 @@ $button-transition: background-color 0.25s ease-out, color 0.25s ease-out !defau
   }
 
   -webkit-appearance: none;
-  border: 1px solid transparent;
```

Comment:
We have enough problems with having to create a variable for every single property of each element, let's not begin to make a _part_ of each property customizable 😉.

Why not `border: $button-border;` ?